From 1e2b60db002577599d97f03ece2df5e2fa5405ab Mon Sep 17 00:00:00 2001
From: bai-cao <2944717202@qq.com>
Date: Sat, 14 Jun 2025 10:51:39 +0800
Subject: [PATCH] update function statement

---
 replace_string.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/replace_string.py b/replace_string.py
index 6364400..7d3fc1a 100644
--- a/replace_string.py
+++ b/replace_string.py
@@ -5,7 +5,12 @@
 import re
 
 def replace_string(olds:str,news:str=""):
-    pass
+    """
+    添加函数说明，以便测试补丁
+    :param olds:
+    :param news:
+    :return:
+    """
     with open("replace.txt","r+",encoding="utf-8") as f:
         txt=f.read()
     replText=re.sub(olds,news,txt) # 替换
-- 
2.49.0.windows.1

